README.md

```diff
@@ -1 +1,62 @@
-# Readme
+# VS Verification Toolbox
+
+This module provides several useful tools for writing VS Code extensions that verify code.
+
+## Features
+
+### General Utilities
+
+The `util` package offers several convenient utilities and wrappers.
+
+#### File System Locations
+
+A `Location` represents a directory on the file system, providing a simple API for navigating through the hierarchy. Example usage:
+
+```typescript
+const root = new Location("/tmp/folder");
+root.basePath; // "/tmp/folder"
+root.enclosingFolder; // Location(/tmp)
+root.child("sub", "path"); // Location(/tmp/folder/sub/path)
+root.path("sub", "path"); // "/tmp/folder/sub/path"
+root.executable("run"); // unix-based: "/tmp/folder/run"; windows: "/tmp/folder/run.exe"
+await root.mkdir(); // ensures the folder exists
```

Comment:
Does this create one folder or all the nested folders mentioned in the path?

Reply:
It calls `fs-extra`'s `ensureDir`, which recursively ensures not only the final folder but also the path leading up to it exists.